src/priorityQueue.js

```diff
@@ -0,0 +1,41 @@
+'use strict'
+
+class Node {
+  constructor(follows, data, priority, proceeds) {
+    this.after = follows
+    this.info = data
+    this.priority = priority
+    this.before = proceeds
+  }
+}
+
+class priorityQueue {
+  constructor() {
+    this.head = null
+    this.tail = null
+    this.count = 0
+  }
+}
+
+  enqueue(value, num) {// adds an element with priority (number) to the back of the queue.
+    let behind = this.tail
+    let inFront = null
+    const element = new Node(behind, value, num, inFront)
+    this.count === 0
+      ?(this.tail = element,
```

Comment:
I understand it's difficult, but only because most people don't use this syntax.
As more people become aware, it will hopefully be used if not at least easily understood. (Similar to fat arrow notation)